pages/how-to/set-up/mgmt-cluster.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Setting Up a New MGMT Cluster
+description: Using Plural CLI to Deploy a MGMT Kubernetes Cluster
+---
+
+### Prerequisites
+[Plural CLI](/how-to/set-up/plural-cli)
+
+##### Ensure Cloud Provider CLI Authentication
```

Comment:
I believe we have some existing docs on how to set up cloud cli authentication as well we should link to or just duplicate, that's a bit better formatted as well

Reply:
linked to the existing cli install